pkg/service/backup/service.go

```diff
@@ -117,215 +118,92 @@ func (s *Service) GetTarget(ctx context.Context, clusterID uuid.UUID, properties
 	s.logger.Info(ctx, "Generating backup target", "cluster_id", clusterID)
 
```

Comment:
This comment refers to the function name.
`GetTarget` is not sufficient here. First of all, it indicates that the method is kind of simple getter (what is not true).
Besides, even if it would be a simple getter, idiomatic go suggests to name getter this way https://go.dev/doc/effective_go#Getters (Target() instead of GetTarget()) .

Please change the name of the method (on the interface level) to sth like `ConvertRunnerPropertiesToTarget()`

Reply:
```go
// BackupService service interface for the REST API handlers.
type BackupService interface {
	GetTarget(ctx context.Context, clusterID uuid.UUID, properties json.RawMessage) (backup.Target, error)
```
I would say that this method is called `GetTarget` because it is the base for the handler of:
```go
m.Get("/{task_type}/target", h.getTarget)
```
and it seems like this is the convention used in all services interfaces:
```go
// ClusterService service interface for the REST API handlers.
type ClusterService interface {
	ListClusters(ctx context.Context, f *cluster.Filter) ([]*cluster.Cluster, error)
	GetCluster(ctx context.Context, idOrName string) (*cluster.Cluster, error)
	PutCluster(ctx context.Context, c *cluster.Cluster) error
	DeleteCluster(ctx context.Context, id uuid.UUID) error
```
@karol-kokoszka do you want me to rename all of them?

Reply:
Ohhh, so the service interface defines HTTP API.
Please leave the GetTarget, but build another method `ConvertRunnerPropertiesToTarget` that returns the target.
Let GetTarget call this new method.